### PR TITLE
Initializing jupyter book

### PR DESCRIPTION
**Pull request by lfunderburk**
Course structure
YAML files to config jupyter book 

Solves #1

### CODE REVIEW

**Comment by lfunderburk** [comment]
Hi @edublancas 

Removed _build directory from any future commits

File: course-material/docs/markdown-notebooks.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.5
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Notebooks with MyST Markdown
```
Comment (edublancas) [inline course-material/docs/markdown-notebooks.md:15]: this looks like a sample file, we can remove it

File: course-material/docs/markdown.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# Markdown Files
```
Comment (edublancas) [inline course-material/docs/markdown.md:1]: same, this looks like a sample file

File: course-material/docs/notebooks.ipynb
Comment (edublancas) [inline course-material/docs/notebooks.ipynb:1]: sample file

File: course-material/references.bib [hunk 3]
```diff
@@ -0,0 +1,56 @@
+---
+---
+
+@inproceedings{holdgraf_evidence_2014,
```
Comment (edublancas) [inline course-material/references.bib:4]: sample file

**Review by edublancas — CHANGES_REQUESTED** [review]
please remove the sample files generated by jupyter-book

Also, I see you have a lot of placeholder notebooks, I'd recommend you converting them into `.md` with jupytext (jupyter supports  .md) as well. once you install jupytext, you can open them as notebooks and execute them. .md is an easier format to manage on git

File: course-material/docs/markdown-notebooks.md (hunk 1 above)
Comment (lfunderburk) [reply course-material/docs/markdown-notebooks.md]: Resolved in latest commit

File: course-material/docs/notebooks.ipynb
Comment (lfunderburk) [reply course-material/docs/notebooks.ipynb:1]: removed all sample files

File: course-material/references.bib (hunk 3 above)
Comment (lfunderburk) [reply course-material/references.bib:4]: removed all sample files in latest commit

**Comment by lfunderburk** [comment]
@edublancas 
The interview posts notebooks were repetitive, so I condensed them into one 

https://github.com/lfunderburk/sql/blob/main/course-material/exploratory-data-analysis-with-sql/absenteeism-dataset.md

**Review by edublancas — APPROVED** [review]

**Pull request merged**